easybuild/easyconfigs/r/rtg-tools/rtg-tools-3.9.1-Java-1.8.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CmdCp'
+
+name = 'rtg-tools'
+version = '3.9.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://www.realtimegenomics.com/products/rtg-tools'
+description = """ RTG Tools contains utilities to easily manipulate and accurately compare multiple VCF files,
+ as well as utilities for processing other common NGS data formats. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/RealTimeGenomics/rtg-tools/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['e4738c236a5a1ad13754500b2078c9500d66051ac305abf96730015230c4f830']
+
+builddependencies = [('ant', '1.10.5', versionsuffix, True)]
+dependencies = [('Java', '1.8', '', True)]
+
+comp_cmd = 'ant runalltests && ant zip-nojre'
+comp_cmd += ' && unzip %(builddir)s/rtg-tools-%(version)s/dist/rtg-tools-%(version)s-unknown-nojre.zip'
+cmds_map = [('.*', comp_cmd)]
+
+extr_dir = 'rtg-tools-%(version)s-unknown'
+files_to_copy = [(['%s/%s' % (extr_dir, x) for x in ('rtg', 'RTG.jar')], 'bin'), extr_dir]
```

Comment:
@smoors Wouldn't it make more sense to copy the entire contents of the unpacked dir, and set `$PATH` to the top-level install dir?